src/upgrade.rs

```diff
@@ -233,6 +231,7 @@ impl fmt::Debug for OnUpgrade {
 #[cfg(any(feature = "http1", feature = "http2"))]
 impl Pending {
     pub(super) fn fulfill(self, upgraded: Upgraded) {
+        #[cfg(any(feature = "http1", feature = "http2"))]
```

Comment:
why was it necessary to add these additional cfg attributes? the entire impl block is only enabled with `#[cfg(any(feature = "http1", feature = "http2"))]`, so AFAICT, this shouldn't be necessary?

Reply:
Good catch. Thank you for pointing this out.